Jenkinsfile

```diff
@@ -404,6 +404,34 @@ pipeline {
                                 '''
                             }
                         }
+                        stage("Unit tests") {
+                            environment {
+                                DOCKER_ROOTLESS = '1'
+                            }
```

Comment:
I don’t think this env is consumed by unit tests

Reply:
I guess we can execute `rootlesskit go test` in hack/make/unit when this env var is set.

rootlesskit flags could be copied from dockerd-rootless.sh